modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionResourceTest.java

```diff
@@ -1160,6 +1164,191 @@ public void testPutObjectDefinition() throws Exception {
 			Assert.assertEquals("BAD_REQUEST", problem.getStatus());
 		}
 
+		// WorkflowDefinitionLink company scope
+
+		WorkflowDefinition workflowDefinition1 =
+			_workflowDefinitionManager.getWorkflowDefinition(
+				WorkflowDefinitionConstants.
+					EXTERNAL_REFERENCE_CODE_SINGLE_APPROVER,
+				TestPropsValues.getCompanyId());
+
+		WorkflowDefinitionLink workflowDefinitionLink1 =
+			new WorkflowDefinitionLink() {
+				{
+					groupExternalReferenceCode = StringPool.BLANK;
+					workflowDefinitionName = workflowDefinition1.getName();
+				}
+			};
+
+		postObjectDefinition = _addObjectDefinition(randomObjectDefinition());
+
+		postObjectDefinition.setWorkflowDefinitionLinks(
+			new WorkflowDefinitionLink[] {workflowDefinitionLink1});
+
+		postObjectDefinition = objectDefinitionResource.putObjectDefinition(
+			postObjectDefinition.getId(), postObjectDefinition);
+
+		com.liferay.portal.kernel.model.WorkflowDefinitionLink
+			serviceBuilderWorkflowDefinitionLink =
+				_workflowDefinitionLinkLocalService.getWorkflowDefinitionLink(
+					TestPropsValues.getCompanyId(), 0,
+					postObjectDefinition.getClassName(), 0, 0, false);
+
+		Assert.assertEquals(
+			workflowDefinition1.getName(),
+			serviceBuilderWorkflowDefinitionLink.getWorkflowDefinitionName());
+
+		// WorkflowDefinitionLink site scope
```

Comment:
Lets use the same postObjectDefinition for the 3 scenarios:
- adding WorkflowDefinitionLinks 
- updating the existing WorkflowDefinitionLinks 
- removing the existing WorkflowDefinitionLinks